library/runtime-core/src/commonMain/kotlin/io/matthewnelson/kmp/tor/runtime/core/address/IPAddress.kt

```diff
@@ -48,6 +64,21 @@ public sealed class IPAddress private constructor(value: String): Address(value)
                 ?: throw IllegalArgumentException("$this does not contain an IP address")
         }
 
+        /**
+         * Converts network ordered bytes to an IPv4 or IPv6 address.
+         *
+         * @return [IPAddress]
+         * @throws [IllegalArgumentException] if array size is not 4 or 16.
+         * */
+        @JvmStatic
+        @JvmName("get")
+        @Throws(IllegalArgumentException::class)
+        public fun ByteArray.toIPAddress(): IPAddress = when (size) {
+            4 -> toIPAddressV4()
+            16 -> toIPAddressV6()
+            else -> throw IllegalArgumentException("Invalid array size[$size]")
```

Comment:
Since no `scope` is being declared here for `IPAddressV6`, can switch to if null then throw.

Reply:
Resolved in commit 874e176